config/_default/menus/main.en.yaml

```diff
@@ -4987,23 +4990,135 @@ menu:
       identifier: data_streams_metrics_and_tags
       parent: data_streams
       weight: 5
-    - name: Data Jobs Monitoring
-      url: data_jobs/
-      pre: data-jobs-monitoring
-      identifier: data_jobs
-      parent: apm_heading
-      weight: 60000
-    - name: Data Observability
+    - name: Data Observability Overview
       url: data_observability/
       pre: inventories
       identifier: data_observability
-      parent: apm_heading
-      weight: 70000
-    - name: Datasets
-      url: data_observability/datasets
-      identifier: datasets
+      parent: data_observability_heading
+      weight: 60000
+    - name: Data Quality
+      url: data_observability/data_quality
+      identifier: data_quality
       parent: data_observability
       weight: 100000
+    - name: Quality Monitoring
+      url: data_observability/quality_monitoring/
+      pre: check-light-wui
+      identifier: quality_monitoring
+      parent: data_observability_heading
+      weight: 70000
+    - name: Data Quality
+      url: data_observability/data_quality
+      identifier: data_quality
+      parent: quality_monitoring
+      weight: 100000
+    - name: Jobs Monitoring
+      url: data_observability/jobs_monitoring/
+      pre: data-jobs-monitoring
+      identifier: data_jobs
+      parent: data_observability_heading
+      weight: 80000
+    - name: Integrations
```

Comment:
@kevinzenghu I see what we're doing here with a single integrations section but I actually think we move these under the respective SKUs so it's clear which product you are setting up billing wise by following  the docs. I think it's confusing clicking on Databricks under job monitoring and being jumped to transormations ourside the job monitoring docs context. I propose we do something like
 
Quality Monitoring
- Warehouses
-- Setup Snowflake
-- Setup BigQuery
- BI 
-- Setup Tableau
Jobs Mointoring
-- Databricks
-- dbt

etc.

Note no other Datadog product has a top level integrations nav item.